internal/clusterinfo/clusterinfo_static.go

```diff
@@ -0,0 +1,52 @@
+//go:build DISABLE
```

Comment:
I know we prefer to use a static client in future but does it really matter to switch?  Looking at this code one particular concern I have is what is the behavior not running on an HCP cluster?  Won't the operator fail out right of the hypershift APIs are not available.  The init() function concerns me for nonHCP scenarios.  I'm betting the dynamic client is not subject to these same concerns

Reply:
+1. I used the static code & tests to give myself some confidence that I got the field names etc. correct in the dynamic code, but if we have to use the dynamic code then there's no real need to switch it unless/until we have to modify the logic in some way.

Reply:
Will clean up - static code is just a distraction now.